packages/camera/camera_web/example/pubspec.yaml

```diff
@@ -21,4 +21,4 @@ dev_dependencies:
     sdk: flutter
   integration_test:
     sdk: flutter
-  mocktail: ^0.3.0
+  mocktail: 0.3.0
```

Comment:
![giphy](https://user-images.githubusercontent.com/1255594/225495149-01de9182-7236-440a-9270-fd104ab3e8ed.gif)

(My bad, this should haven't happened in the first place)